app/views/peeps.erb

```diff
@@ -0,0 +1,23 @@
+<div class='posting'>
+  <h2>What's happening?</h2>
+  <form id='peeping' action='/peeps/new' method='post'>
+    <textarea class="message form-control" form='peeping' rows="2" maxlength="140" name='peep' required></textarea>
+    <br>
+    <input class="btn btn-default navbar-btn" type='submit' value='Peep'>
+    <span class="countdown"></span>
```

Comment:
I'd been hoping to add a character countdown to the textarea box (as is the case in the real Twitter) - but did a bit of research and realised that it needed JQuery and never got round to it. If time had not been a limiting factor, I'd have liked to figure this out.